src/MSBuild.UnitTests/TerminalLogger_Tests.cs

```diff
@@ -233,20 +234,20 @@ private BuildErrorEventArgs MakeErrorEventArgs(string error)
 
         #region Build summary tests
 
-        private void InvokeLoggerCallbacksForSimpleProject(bool succeeded, Action additionalCallbacks)
+        private void InvokeLoggerCallbacksForSimpleProject(bool succeeded, string projectFile, Action additionalCallbacks)
         {
```

Comment:
Consider a maybe-more-ergonomic change

```suggestion
        private void InvokeLoggerCallbacksForSimpleProject(bool succeeded, Action additionalCallbacks, string projectFile = null)
        {
            projectFile ??= _projectFile;
```

(then you wouldn't have to make the other callsites more complicated)